app/components/addPlayList.tsx

```diff
@@ -0,0 +1,75 @@
+"use client";
+import { useState } from "react";
+import { addPlayList } from '../lib/api-service';
+import Link from "next/link";
+
+export default function AddPlayList() {
+    const [name, setName] = useState('');
+    const [movies, setMovies] = useState<Array<string>>(['']);
+
+    const handleSubmit = async (e: any) => {
+        e.preventDefault();
+        const data = await addPlayList({name: name, movies: movies});
+        if (data?.id) {
+            alert('Playlist added successfully');
+        } else {
+            alert('Failed to add playlist');
+        }
+    }
+
+    const handleChange = (e: any, i: number) => {
+        const {value}=e.target
+        const onchangeVal = [...movies]
```

Comment:
can we rename this to variable name like updatedMovies right now looks like function name

Reply:
updated variable name